test/controller.bar.tests.js

```diff
@@ -251,10 +251,10 @@ describe('Bar controller tests', function() {
 
 		var meta0 = chart.getDatasetMeta(0);
 
-		[	{ b: 290, w: 91, x:  95, y: 161 },
-			{ b: 290, w: 91, x: 209, y: 419 },
-			{ b: 290, w: 91, x: 322, y: 161 },
-			{ b: 290, w: 91, x: 436, y: 419 }
+		[{b: 290, w: 91, x: 95, y: 161},
```

Comment:
I think that would be the best way to do it, I will update the PR with that change soon.